CMakeLists.txt

```diff
@@ -16,10 +16,14 @@ target_link_libraries(net
 add_executable(roboteam_world
         src/main.cpp
         src/WorldHandler.cpp
-        src/kalman/KalmanFilter.cpp
-        src/kalman/KalmanObject.cpp
-        src/kalman/KalmanBall.cpp
-        src/kalman/KalmanRobot.cpp)
+        src/kalman/WorldFilter.cpp
+        src/util/KalmanFilter.h
+        src/kalman/RobotFilter.cpp
+        src/util/Scaling.h
+        src/kalman/BallFilter.cpp
```

Comment:
I find the folder structure a little bit confusing. There is a kalman folder with contains filters, and a util folder which contains kalmanfilter. It's not much so maybe just don't divide it into folders?

Reply:
Yeah I still need to restructure the repo (also for linking it as library etc.)